### PR TITLE
Email Dashboard

### PR DESCRIPTION
**Pull request by acarb95**
This contains the email dashboard, I haven't been able to build it yet. I created it on a QA machine (119), so it should work, assuming I didn't make a typo deleting the stuff up to source.

Right now it contains the "Attach" column in the table. That displays a True or False value. When Alex gets the downloading done, it will display a checkbox where they user can download the attachment.

The Sender Domain graph doesn't show anything right now, I think there is a PR up to add the field back. So once that is merged the Sender Domain graph will work. 

If someone could pull it down and test it, that would be great. Already got it passed by Rob.

### CODE REVIEW

**Comment by craig-cogdill** [comment]
Comments from stand up with Matt:

- Emails by Receiver changes to the AttachType widget from the FileReconstruction dashboard
- Making the SMTP line-graph bigger on the left and moving the Table search below all the charts

File: resources/visualizations/Top-10-Attachment-Types-By-Count.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "title": "Top 10 Attachment Types By Count",
```
Comment (KjellKod) [inline resources/visualizations/Top-10-Attachment-Types-By-Count.json]: tx

Comment (KjellKod) [inline resources/visualizations/Top-10-Attachment-Types-By-Count.json]: @Rob, would we be interested for (3.1.2. to get "Bottom 10 Attachment Types By Count"?

Comment (yannick-polius) [inline resources/visualizations/Top-10-Attachment-Types-By-Count.json]: @KjellKod Rob agrees for later

**Comment by KjellKod** [comment]
:+1:

**Comment by KjellKod** [comment]
:+1:  from Rob

**Pull request merged**